Bookish/Controllers/BookController.cs

```diff
@@ -152,8 +177,45 @@ public async Task<ActionResult> BookInput(BookInput input)
         }
         return View();
     }
+
+    [HttpGet]
+    public async Task<ActionResult> BookEdited(Book input)
+    {
+        string bookId = input.Id;
+        using (var context = new EFCore())
+        {
+            var book = context.Books.Find(bookId);
+            if (book != null)
+            {
+                if (input.Title != null) book.Title = input.Title;
+                if (input.Author != null) book.Author = input.Author;
+                context.SaveChanges();
+                return View();
+            }
+
+            return View("BookNotEdited");
+        }
+    }
     
-    
+    [HttpGet]
+    public async Task<ActionResult> BookDeleted(Book input)
```

Comment:
Similar comments: make method name describe what it does, and consider if this is really a GET endpoint and not something else